src/ui/pages/Networks/Networks.tsx

```diff
@@ -222,23 +227,23 @@ function NetworkPage() {
   const mutation = useMutation({
     mutationFn: saveChainConfig,
     onSuccess() {
-      networksStore.update();
+      updateNetworks();
```

Comment:
should we add `await` here?

Reply:
wasn't here before so 🤷‍♀️ and unless it's needed, it's preferred no to delay